projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/JobExecutionRepository.java

```diff
@@ -38,4 +38,12 @@
    List<DataJobExecutionIdAndEndTime> findByDataJobNameAndStatusNotInOrderByEndTime(String jobName, List<ExecutionStatus> statuses);
 
    List<DataJobExecution> findDataJobExecutionsByStatusInAndStartTimeBefore(List<ExecutionStatus> statuses, OffsetDateTime startTime);
+
+   @Query("SELECT dje.status AS status, dje.dataJob.name AS name, count(dje.status) AS count " +
+          "FROM DataJobExecution dje " +
+          "WHERE dje.status IN :statuses " +
+          "AND dje.dataJob.name IN :dataJobs " +
+          "GROUP BY dje.status, dje.dataJob")
```

Comment:
Can you add the actual query generated by hibernate to the description or as a comment.

Reply:
`Hibernate: select datajobexe0_.status as col_0_0_, datajobexe0_.job_name as col_1_0_, count(datajobexe0_.status) as col_2_0_ from data_job_execution datajobexe0_ where (datajobexe0_.status in (? , ?)) and (datajobexe0_.job_name in (?)) group by datajobexe0_.status , datajobexe0_.job_name
`

Reply:
Thanks